Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -773,7 +773,10 @@ bool CSSPropertyParser::consumeFontVariantShorthand(bool important)
     do {
         if (!capsValue && (capsValue = CSSPropertyParsing::consumeFontVariantCaps(m_range)))
             continue;
-        
+
+        if (m_range.peek().id() == CSSValueNormal)
+            return false;
+
```

Comment:
Should we make this the first check?

Reply:
Yes it is best to put the check first so that we can just return false before having to do the capsValue check, which returns false for "normal". 

For correctness though, it doesn't make a difference as long as we keep this check above `if (!positionValue && (positionValue = CSSPropertyParsing::consumeFontVariantPosition(m_range)))
            continue;` 
   which will return true for "normal". That was what was happening before.